atom/browser/api/atom_api_web_contents.h

```diff
@@ -6,13 +6,17 @@
 #define ATOM_BROWSER_API_ATOM_API_WEB_CONTENTS_H_
 
 #include <string>
+#include <vector>
+#include <set>
```

Comment:
Yeah it should be moved, `api_web_contents.h` is also included by other files, adding a new include in `api_web_contents.h` would slow down compilation speed for all those files. Even though it only slows down a little, it is still a good habit to keep minimal `include`s.